### PR TITLE
making sure launcher is listed in central

### PR DESCRIPTION
**Pull request by adietish**

### CODE REVIEW

**Comment by adietish** [comment]
@jeffmaury dare to review and test please?

File: devstudio/com.jboss.devstudio.central.discovery/plugin.xml [hunk 1]
```diff
@@ -207,7 +207,26 @@ reasonable, reporting issues to these providers as required.</description>
...
          <overview
-               url="http://tools.jboss.org/">
+               url="http://tools.jboss.org/features/openshift.html">
+         </overview>
+      </connectorDescriptor>
+
+      <connectorDescriptor
+            categoryId="org.jboss.tools.central.discovery.a.web"
+            groupId="org.jboss.tools.central.discovery.a.web.core"
+            description="Provides Tooling for Red Hat's Fabric 8 Launcher."
```
Comment (nickboldt) [inline devstudio/com.jboss.devstudio.central.discovery/plugin.xml]: certificationId="com.jboss.devstudio.discovery.certification.supported"
is missing

Comment (nickboldt) [inline devstudio/com.jboss.devstudio.central.discovery/plugin.xml]: Should this be Fabric8, not Fabric 8 ?

Comment (adietish) [reply devstudio/com.jboss.devstudio.central.discovery/plugin.xml]: @nickboldt yes, Fabric8 like it is referred to here: http://fabric8.io/
> Rest of fabric8 is

**Pull request merged**